package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@rev_ai/revai-node-sdk",
-  "version": "1.0.0",
+  "name": "revai-node-sdk",
+  "version": "1.0.2",
```

Comment:
I think we're on version 1.0.4 now, we'll need to increase the version, if we want to deploy again.

Reply:
Actually, I didn't realize this was the package-lock. Why is this included the package lock file?

Reply:
Package-lock doesn't actually get transferred over when we deploy. I'm not sure if we can fully remove the file but its effectively dead code. Only the version in package.json matters

Reply:
Usually this is done by npm cli when you run npm uninstall package-name. I think you should try running this command to fix the package-lock